Source/ZoomNet/Resources/Webinars.cs

```diff
@@ -94,11 +94,12 @@ public Task<PaginatedResponseWithToken<Webinar>> GetAllAsync(string userId, int
 		/// <param name="settings">Webinar settings.</param>
 		/// <param name="trackingFields">Tracking fields.</param>
 		/// <param name="cancellationToken">The cancellation token.</param>
+		/// <param name="templateId">Template Identifer.</param>
 		/// <returns>
 		/// The new webinar.
 		/// </returns>
 		/// <exception cref="System.Exception">Thrown when an exception occured while creating the webinar.</exception>
-		public Task<ScheduledWebinar> CreateScheduledWebinarAsync(string userId, string topic, string agenda, DateTime start, int duration, string password = null, WebinarSettings settings = null, IDictionary<string, string> trackingFields = null, CancellationToken cancellationToken = default)
+		public Task<ScheduledWebinar> CreateScheduledWebinarAsync(string userId, string topic, string agenda, DateTime start, int duration, string password = null, WebinarSettings settings = null, IDictionary<string, string> trackingFields = null, CancellationToken cancellationToken = default, string templateId = null)
```

Comment:
See my previous comment regarding position of the templateId parameter.

Reply:
Great suggestion - I also agree (and should have spotted this). I've updated the code.